src/lib/reducers/apps.js

```diff
@@ -11,19 +11,28 @@ const EXCLUDES = ['settings', 'onboarding']
 const CATEGORIES = ['cozy', 'partners', 'ptnb']
 const DEFAULT_CATEGORY = 'others'
 
+const isCurrentApp_ = state => app =>
+  app.name === state.appName && app.name_prefix === state.appNamePrefix
+
 // selectors
-export const getApps = state => {
-  if (__TARGET__ !== 'mobile') {
-    return state.apps && state.apps.data
+const onMobile = __TARGET__ !== 'mobile'
+export const getApps = (state, mobile = onMobile) => {
+  if (!state.apps) return []
+
+  if (!mobile) {
+    return state.apps
   }
 
-  if (!state.apps || !state.apps.data) return []
-  return state.apps.data.filter(app =>
```

Comment:
Is normal thate here you don't use the data property anymore? I don't see it anywhere else

Reply:
It is normal the apps.data was not useful and was complicating the code. The `apps` from the action are directly stored under the `apps` property of this slice.